tests/test_json_operations.py

```diff
@@ -25,20 +25,26 @@
 
 # Test that the file is actually created
 def test_json_creation():
-    output_path = "./tests/output_test_files/test_json_creation.json"
+    output_path = "./output_test_files/test_json_creation.json"
+    if "TOX_ENV_NAME" in os.environ:
```

Comment:
Extract into a method with a comment explaining the pycharm vs tox location problem